### ISSUE TITLE
Disable Traefik through Rancher Desktop CLI (rdcli)

### ISSUE THREAD

**Issue by royteeuwen** [rancher-sandbox/rancher-desktop#3281]
### Problem Description

I am trying to automate the setup of rancher desktop for all developers on their local machines so that everything is available and reflects the clusters we have in the cloud. This means we would like to use the nginx ingress controller instead of traefik. I would like to be able to disable Traefik through the cli and also be able to tell when the kubernetes cluster is up and running again without traefik

### Proposed Solution

Provide a flag to set the Traefik with `rdcli set`

**Comment by jandubois** [comment]
Yes, this needs to be exposed through `rdctl start` and `rdctl set`.

As a workaround you can do this:

```console
$ rdctl api -X PUT --body '{"kubernetes":{"options":{"traefik":false}}}' settings
reconfiguring Rancher Desktop to apply changes (this may take a while)
```

Note that this is calling the internal API directly, and subject to change, so please update your automation once the option is available through `start` and `set`!

**Comment by jandubois** [comment]
This is available in Rancher Desktop 1.8 using

```
rdctl set --kubernetes.options.traefik=false
```
